app/components/Skills/Skills.scss

```
@@ -5,6 +5,20 @@ ul {
   text-align: center;
 }
 
+.skills-list-item {
+  line-height: 1.7;
+}
+
 .skill > a {
   @include link;
+  display: inline-flex;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)